fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/PropertiesRdfContext.java

```diff
@@ -76,7 +82,7 @@ public PropertiesRdfContext(final FedoraResource resource,
         }
 
         final UnmodifiableIterator<Property> properties =
-                Iterators.filter(allProperties, isInternalProperty.negate()::test);
+                Iterators.filter(allProperties, and(isInternalProperty.negate()::test, not(IS_UUID)));
```

Comment:
After the Java 8 PRs, perhaps better `isInternalProperty.and(IS_UUID.negate())::test`, or even change `IS_UUID` to `IS_NOT_UUID` and remove the negation here.